products/sle/main.pm

```diff
@@ -458,6 +458,9 @@ sub load_inst_tests() {
             loadtest 'installation/partitioning_full_lvm';
         }
         if (get_var("FILESYSTEM")) {
+            if (get_var('PARTITIONING_WARNINGS')) {
+                loadtest 'installation/partitioning_warnings';
+            }
```

Comment:
@okurz the Factory first policy states that code reaching SLE should already be in Tumbleweed before it's submitted to SLE

Therefore if the feature is ready, then it shouldn't need a second validation run on openSUSE

Sure..it if breaks, that would imply SUSE's policies haven't been adhered to, well then you can escalate that to both a SLE release manager and your friendly openSUSE Chairman